app/helpers/value_helper.rb

```diff
@@ -24,11 +24,16 @@ def decimal?(val)
   end
 
   def property_to_s(val)
-    # assume all decimals are currency
-    if decimal?(val)
+    if decimal?(val) # assume all decimals are currency
       number_to_currency(val)
+    elsif val.is_a?(ActiveSupport::TimeWithZone)
+      I18n.l(val, format: :date)
+    elsif val == true
+      "Yes"
+    elsif val == false
+      "No"
     else
-      val.to_s
+      val
     end
```

Comment:
Well, it wouldn't work for the `decimal?` call, but for the rest, just saves you writing `val == ` or `val.is_a?` over and over.

```ruby
case val
when ActiveSupport::TimeWithZone
  I18n.l(val, format: :date)
when true
  "Yes"
when false
  "No"
else
  val
end
```

Not a big deal!